content/en/docs/languages/net/exporters.md

```diff
@@ -105,12 +108,35 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     // Other setup code, like setting a resource goes here too
 
     .Build();
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddOtlpExporter(options =>
+    {
+        options.Endpoint = new Uri("your-endpoint-here/v1/metrics");
+        options.Protocol = OtlpExportProtocol.HttpProtobuf;
+    })
+
+    // Other setup code, like setting a resource goes here too
+
+    .Build();
+
+using var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddOpenTelemetry(logging =>
+    {
+        logging.AddOtlpExporter(options =>
+        {
+            options.Endpoint = new Uri("your-endpoint-here/v1/logs");
+            options.Protocol = OtlpExportProtocol.HttpProtobuf;
+        })
+    });
+});
 ```
 
 Use environment variables to set values like headers and an endpoint URL for
 production.
 
-### Note for .NET Core 3.1 and below and gRPC
+#### Note for .NET Core 3.1 and below and gRPC
```

Comment:
Given .NET Core 3.1 is out-of-support, and not supported by OTel .NET, lets remove it.

Reply:
removed